tests/unit/PackageTest.php

```diff
@@ -1458,8 +1465,10 @@ function (\Throwable $throwable) use ($package): void {
             ->whenHappen(
                 function (\Throwable $throwable) use ($exception, $package): void {
                     $this->assertThrowableMessageMatches($throwable, 'boot application');
+                    /** @var \Throwable $previous */
                     $previous = $throwable->getPrevious();
```

Comment:
Instead of overwriting the type, which may be incorrect, we could add another expectation:

```suggestion
                    $previous = $throwable->getPrevious();
                    static::assertTrue($previous instanceof \Throwable);
```

This is better, and will actually catch if there is a case where there was no previous `Throwable` instance.

Do this for the following two instances as well.